src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -22,46 +22,59 @@ import { ALL_USE_CASE_ID } from '../../../../../core/utils';
 
 export interface CollapsibleNavTopProps {
   navLinks: ChromeNavLink[];
-  navGroupsMap: Record<string, NavGroupItemInMap>;
   currentNavGroup?: NavGroupItemInMap;
   navigateToApp: InternalApplicationStart['navigateToApp'];
   logos: Logos;
   onClickBack?: () => void;
   onClickShrink?: () => void;
   shouldShrinkNavigation: boolean;
+  visibleUseCases: NavGroupItemInMap[];
 }
 
 export const CollapsibleNavTop = ({
   navLinks,
-  navGroupsMap,
   currentNavGroup,
   navigateToApp,
   logos,
   onClickBack,
   onClickShrink,
   shouldShrinkNavigation,
+  visibleUseCases,
 }: CollapsibleNavTopProps) => {
   const homeLink = useMemo(() => navLinks.find((link) => link.id === 'home'), [navLinks]);
 
-  const shouldShowBackButton = useMemo(
-    () =>
-      currentNavGroup?.id !== ALL_USE_CASE_ID &&
-      !shouldShrinkNavigation &&
-      Object.values(navGroupsMap).filter((item) => !item.type).length > 1 &&
-      currentNavGroup,
-    [navGroupsMap, currentNavGroup, shouldShrinkNavigation]
+  const isInGlobalWorkspace = useMemo(
```

Comment:
What is global workspace referring? Shall we not use the term "global workspace" as we no longer have this?

Reply:
Sure, change to isOutsideWorkspace.